generate/avro_template.go

```diff
@@ -93,6 +130,33 @@ func Test{{ .name }}_WriteAvroCF(t *testing.T) {
 		t.Error("OCF reports deleted true expected false")
 	}
 }
+
+func Example{{ .name }}BulkAvroWrite() {
+	input := []*{{ .name }}{ {}, {}, {} }
+	inputChan := make(chan *{{ .name }})
+
+	devnull, _ := os.Open("/dev/null")
```

Comment:
err?

Reply:
definitely but given this was an example and for example functions there is no return and there is value in keeping them shorter to just show the framework I skipped it.